tests/test_relation_u.py

```diff
@@ -50,7 +50,13 @@ def test_invalid_join():
         rel = dj.U('language') * dict(language="English")
 
     def test_aggregations(self):
```

Comment:
I think we should have tests for `aggregate` and `aggr` methods on non `dj.U` as well.